mautrix/bridge/commands/admin.py

```diff
@@ -29,3 +29,92 @@ async def set_power_level(evt: CommandEvent) -> EventID:
     except (MatrixRequestError, IntentError):
         evt.log.exception("Failed to update power levels")
         return await evt.reply("Failed to update power levels (see logs for more details)")
+
+@command_handler(needs_admin=True, needs_auth=False, name="set-avatar",
+                 help_section=SECTION_ADMIN, help_args="<_mxc_uri_> [_mxid_]",
```

Comment:
The help texts are for users, so they don't need to match variable names

```suggestion
                 help_section=SECTION_ADMIN, help_args="<_mxc:// uri_> [_mxid_]",
```

Reply:
Copy and paste oversight